### ISSUE TITLE
Support multi-line queries in CLI

### ISSUE THREAD

**Issue by ryzhyk** [feldera/feldera#2507]
The usual behavior for a SQL CLI is that if you press Enter  and the line doesn't end in a semicolon, it enters the multiline editing mode. FDA currently attempts to parse the line as is and will return an error if it's not a complete statement.

**Comment by ryzhyk** [comment]
yep, pasting works, and you can edit the pasted multi-line statement before hitting Enter.

**Comment by mihaibudiu** [comment]
It works with shift-enter.
